authorization/Dockerfile

```
@@ -1,28 +1,33 @@
 
 COPY gradlew .
 COPY gradle gradle
-COPY settings.gradle .
 COPY auth-shared auth-shared
 COPY authorization authorization
 
+RUN echo "include 'authorization'" > settings.gradle && \
+    echo "include 'auth-shared'" >> settings.gradle
+
 RUN chmod +x gradlew
 RUN ./gradlew :authorization:clean :authorization:build -x test --no-daemon
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)